From ced6e8e51baab4d26aef0e65d917974fea9a370a Mon Sep 17 00:00:00 2001
From: Quintin Hill <Quintin.Hill@arqiva.com>
Date: Tue, 22 Aug 2017 12:18:59 +0100
Subject: [PATCH 2/6] submit: add detail to error message

It is helpful to have the failing file name here too.
---
 src/condor_submit.V6/submit.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/condor_submit.V6/submit.cpp b/src/condor_submit.V6/submit.cpp
index 57d29a517c..95895ef314 100644
--- a/src/condor_submit.V6/submit.cpp
+++ b/src/condor_submit.V6/submit.cpp
@@ -812,8 +812,8 @@ main( int argc, const char *argv[] )
 		submit_hash.insert_source("<stdin>", FileMacroSource);
 	} else {
 		if( (fp=safe_fopen_wrapper_follow(cmd_file,"r")) == NULL ) {
-			fprintf( stderr, "\nERROR: Failed to open command file (%s)\n",
-						strerror(errno));
+			fprintf( stderr, "\nERROR: Failed to open command file (%s) (%s)\n",
+						cmd_file, strerror(errno));
 			exit(1);
 		}
 		// this does both insert_source, and also gives a values to the default $(SUBMIT_FILE) expansion
-- 
2.13.5

